api/envoy/config/filter/http/adaptive_concurrency/v2alpha/adaptive_concurrency.proto

```diff
@@ -7,5 +7,49 @@ option java_outer_classname = "AdaptiveConcurrencyProto";
 option java_multiple_files = true;
 option go_package = "v2alpha";
 
+import "google/protobuf/duration.proto";
+import "google/api/annotations.proto";
+import "google/protobuf/wrappers.proto";
+
+import "validate/validate.proto";
+
+message GradientControllerConfig {
+  // The percentile to use when summarizing aggregated samples. Defaults to p50.
+  google.protobuf.DoubleValue sample_aggregate_percentile = 1
```

Comment:
Did you consider using `envoy.type.Percent`?

Reply:
Representing a p999 seems less awkward via `0.999` than the fractional percent in percent.proto. It should be trivial to swap if you have a strong preference.

Reply:
`Percent` is different from `FractionalPercent`. I would use `Percent` here per @htuch